docs.json

```diff
@@ -123,14 +123,15 @@
                     "5. Hosting", "/widgetbook-cloud/hosting"
                 ],
                 [
-                    "6. Review (beta)", "/widgetbook-cloud/review"
+                    "7. Github Integration", "/widgetbook-cloud/integrating-github"
```

Comment:
While you have to install the GitHub app to make reviews work, I believe it's better to understand for the user when we first introduce reviews and then follow with the GitHub integration.

So, I would change the order so that 6. Reviews, 7. GitHub Integration, 8. Accessing resources.

What do you think?